addons/website_forum/controllers/website_forum.py

```diff
@@ -107,12 +108,14 @@ def questions(self, forum, tag=None, page=1, filters='all', my=None, sorting=Non
 
         if not sorting:
             sorting = forum.default_order
+        values = self._prepare_user_values(forum=forum, searches=post, header={'ask_hide': not forum.active})
```

Comment:
what is `ask_hide` ?

Reply:
something I don't need anymore, I got rid of this part :sweat_smile: 
Also: I corrected everything you pointed out, but during rebase I may have made an oopsie that may have caused closure of this draft pull request :sweat: could you check this draft pull request instead? https://github.com/odoo-dev/odoo/pull/2675  Sorry for the trouble and confusion :sweat: